docs/openapi.yaml

```diff
@@ -39,29 +38,21 @@ components:
       properties:
         sourceType:
           type: string
-          enum: ['arweave', 'otherSource']
-          description: The source type of the evaluation.
+          enum: ['arweave', 'other']
         internalWrites:
           type: boolean
-          description: Flag to enable or disable internal writes.
         useKVStorage:
           type: boolean
-          description: Indicates if key-value storage is used.
         remoteStateSyncEnabled:
           type: boolean
-          description: Flag for enabling remote state synchronization.
         waitForConfirmation:
           type: boolean
-          description: Whether to wait for confirmation before proceeding.
         updateCacheForEachInteraction:
```

Comment:
Just a note, this option is considered deprecated in DRE nodes, might want to remove https://discord.com/channels/1103577987007397919/1105203454495969390/1176114137550573578

Reply:
removed in [8dcbd08](https://github.com/ar-io/arns-service/pull/76/commits/8dcbd0809fe585dc2fd8d04acbfbf768742175a7)